airbyte-integrations/connectors/source-facebook-marketing/integration_tests/conftest.py

```diff
@@ -6,12 +6,15 @@
 import json
 
 import pytest
+from source_facebook_marketing.config_migrations import MigrateAccountIdToArray
 
 
 @pytest.fixture(scope="session", name="config")
 def config_fixture():
     with open("secrets/config.json", "r") as config_file:
-        return json.load(config_file)
+        config = json.load(config_file)
+        migrated_config = MigrateAccountIdToArray.transform(config)
+        return migrated_config
```

Comment:
Is this necessary? I thought the source would perform the migration wherever we pass the config fixture into it

Reply:
Migration is in the main.py, not the source:
https://github.com/airbytehq/airbyte/pull/33538/files#diff-1f20186eea3ef91ef3ce1053eb1dc02fd8ead91b098524ab68af68ae1e15f445R13-R15

Reply:
If this was for unit tests this would make sense to me, but this is for cat tests which spin up a source container and run it - wouldn't that run main.py?

Reply:
This fixture is passed directly to the source object without the `main.py`. https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-facebook-marketing/integration_tests/test_streams.py#L29